src/components/v2/values/chartValuesDiff/ChartValuesView.tsx

```diff
@@ -920,22 +920,23 @@ function ChartValuesView({
     }
 
     const renderValuesTabs = () => {
+        const initialSelectedTab =
+            presetValueId || isCreateValueView
+                ? 'yaml'
```

Comment:
Please use constant. And try to simplify this condition

Reply:
done